models/reddit_ads.yml

```diff
@@ -83,10 +83,10 @@ models:
             - campaign_id
             - currency
     columns:
-      - name: date_day
-        description: '{{ doc("date_day") }}'
-      - name: ad_id
-        description: '{{ doc("ad_id") }}'
+      - name: date_day 
+        description: '{{ doc("date_day") }}' 
+      - name: ad_id 
+        description: '{{ doc("ad_id") }}' 
```

Comment:
Should we apply not_null tests to these fields?

Reply:
updated